CHANGELOG.md

```diff
@@ -2,9 +2,29 @@
 
 Friendly reminder: We have a [bug bounty program](https://hackerone.com/cosmos).
 
+## v0.35.7
+
+June 16, 2022
+
+### BUG FIXES
+
+- [p2p] [\#8692](https://github.com/tendermint/tendermint/pull/8692) pass maxConns for MaxPeers during node setup (#8684)
```

Comment:
```suggestion
- [p2p] [\#8692](https://github.com/tendermint/tendermint/pull/8692) scale the number of stored peers by the configured maximum connections (#8684)
```

Or words to that effect, since they don't really map 1:1.

Reply:
👍🏼 I just copied the text that was in the master changelog. 😆